README.md

```diff
@@ -65,7 +65,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18.x]
+        node-version: [20.x]
```

Comment:
Since SPFx doesn't support Node@20 yet, we should stick to 18 here.

Reply:
I was a little bit too fast there indeed 😊